docs/content/Upgrade/automatedupgrade.md

```diff
@@ -104,10 +104,18 @@ Previously, the existing cluster upgrade focused on updating a cluster's underly
 
 The automated upgrade process provides a mechanism where, instead of being deleted, the existing PostgreSQL clusters will be left in place during the PostgreSQL Operator upgrade. While normal Operator functionality will be restricted on these existing clusters until they are upgraded to the currently installed PostgreSQL Operator version, the pods, services, etc will still be in place and accessible via other methods (e.g. kubectl, service IP, etc).
 
-To upgrade a particular cluster, use
+To upgrade a particular, non-PostGIS cluster, use
 ```    
 pgo upgrade mycluster
 ```
+
+To upgrade a particular, a PostGIS enabled cluster, use
+```    
+pgo upgrade --post-gis-image-tag=centos7-11.8-2.5-4.4.0 mygiscluster
```

Comment:
We now have a new set of params!

```suggestion
pgo upgrade --post-gis-image-tag={{< param centosBase >}}-{{< param postgresVersion >}}-3.0-{{< param operatorVersion >}} mygiscluster
```

(though likely we'll want to add one for `postgisVersion` now.

Reply:
Sounds good, I've updated these references accordingly and added the new postgisVersion param. Thanks!